<?php

namespace ContainerMij7MjT;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_VVC90AfService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.vVC90Af' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.vVC90Af'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'manager' => ['services', 'doctrine.orm.default_entity_manager', 'getDoctrine_Orm_DefaultEntityManagerService', false],
            'user' => ['privates', '.errored..service_locator.vVC90Af.App\\Entity\\User', NULL, 'Cannot autowire service ".service_locator.vVC90Af": it references class "App\\Entity\\User" but no such service exists.'],
            'userPasswordHasher' => ['privates', 'security.user_password_hasher', 'getSecurity_UserPasswordHasherService', true],
        ], [
            'manager' => '?',
            'user' => 'App\\Entity\\User',
            'userPasswordHasher' => '?',
        ]);
    }
}
